internal/stores/README.md

```diff
@@ -4,8 +4,13 @@ Stores are the interface by which metadata is managed and files are uploaded/ret
 ## Upload
 When uploading files, the following items should be handled in the implementation:
 
-* Metadata should be generated from the file and written alongside the binary on disk
+* Loop through the objects you want to upload.
```

Comment:
This seems more like inline comment documentation or a docstring than a README. READMEs should be things the user is concerned with.

If the code is tricky, we should do an inline comment. Another possibility is that the code isn't clear enough to figure this out by looking at it, which is a code smell of its own.

I think its fairly clear by looking at the code, so maybe put this summary as part of the docstring vs inline comment.